server/emails/mailer.tsx

```diff
@@ -56,6 +56,63 @@ export class Mailer {
     }
   }
 
+  template = ({
```

Comment:
In the end I'm not clear on why this customization is needed, it seems to just remove previewText and pass it into `EmailLayout` instead – but it seems unrelated to the right of the work, can you clarify?

Reply:
It puts `bodyContent` in the right place. [Refer the original template](https://github.com/revivek/oy/blob/master/src/utils/HTML4.js), which `bodyContent` further inside the nested structure. It's desirable for `bodyContent` to be the immediate child of `<body>` in our case since it contains the markup under `<script>`

Reply:
Can you add that link to the default template as a code comment, thanks